tests/unit/test_apigateway.py

```diff
@@ -513,3 +515,36 @@ def test_construct_invocation_event(self):
                 t["is_base64_encoded"],
             )
             assert result == t["expected"]
+
+
+class TestRequestParameterResolver:
+    def test_resolve_request_parameters(self):
```

Comment:
I guess that's more of an internal test. Preferably, we have some snapshot tests covering the e2e API.

Reply:
unit test, yea. There are integration tests that use will use this, pretty much any backend integration and that can be snapshotted